### ISSUE TITLE
Error requesting resources from different KBS': The resource KBS host differs from KBS URL one

### ISSUE THREAD

**Issue by katexochen** [confidential-containers/guest-components#183]
When requesting two resources from different KBS through the API of the AA, the second request will fail with:

```
AA-KBC get resource failed: The resource KBS host 10.0.100.1:8080 differs from the KBS URL one 10.0.100.100:8080
```

Can anyone explain to me why we're not just using the `KbsUri` from the request? https://github.com/confidential-containers/attestation-agent/blob/aa1d3c510350cd2f2668aca374abba19e2b73b3f/kbc/src/cc_kbc/mod.rs#L76-L104

**Comment by fitzthum** [comment]
Yes, in that PR @Xynnn007 moved the formulation of the URL from the KBC to the KBS Client. Now it is created [here](https://github.com/confidential-containers/guest-components/pull/304/commits/bc24e54aeadfc716a4eb6b8f3ee790b4c9541f81#diff-baad0b2286cbd2cbb6b4b8d6947b5dfd489da47729ccaad15e589bcd1517e4deR147). There is no longer an error message but we still use the KBS URI from the AA configuration, ignoring what is set in the resource URI. This is correct imo.
